tests/apis.js

```diff
@@ -0,0 +1,430 @@
+import test from 'ava';
+import uuid from 'uuid';
+import ipsum from 'lorem-ipsum';
+import Promise from 'bluebird';
+import slugify from 'underscore.string/slugify';
```

Comment:
I searched for slugify as that's what I was expecting. I can swap to `kebabCase`